charts/jellyfin/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
We should probably standardize this file across the charts somehow... this one is missing the `.vscode` IDE folder

Reply:
I agree, my copy pasta is failing me yet again